Code/GraphMol/RWMol.cpp

```diff
@@ -644,6 +644,11 @@ void RWMol::beginBatchEdit() {
 void RWMol::commitBatchEdit() {
   if (!(dp_delBonds || dp_delAtoms)) {
     return;
+  } else if (dp_delBonds->none() && dp_delAtoms->none()) {
+    // no need to reset, since nothing is removed
```

Comment:
I don't understand the comment here.
You say "no need to reset", but then you reset.

Shouldn't you just be returning?